samples/extensions/color_write_enable/color_write_enable.cpp

```diff
@@ -539,15 +539,11 @@ void ColorWriteEnable::on_update_ui_overlay(vkb::Drawer &drawer)
 {
 	if (drawer.header("Background color"))
 	{
-		if (drawer.slider_float("Red", &background_r_value, 0.0f, 1.0f))
-		{
-			build_command_buffers();
-		}
-		if (drawer.slider_float("Green", &background_g_value, 0.0f, 1.0f))
-		{
-			build_command_buffers();
-		}
-		if (drawer.slider_float("Blue", &background_b_value, 0.0f, 1.0f))
+		if (drawer.color_picker("", 3, background_color.data(), 200,
```

Comment:
With a width of 200 the color picker is a bit too narrow to display the selcted color value. Maybe use a width of 300, instead.

Reply:
I changed it to 300. On my screen, 200 pixels looks fine. You can see it in the screenshot I added to the initial message. Maybe the problem is DPI.